src/applications/login/components/SignInButtons.jsx

```diff
@@ -11,67 +11,92 @@ function loginHandler(loginType) {
       >
-        <img
-          aria-hidden="true"
-          role="presentation"
-          alt="My HealtheVet"
-          src={`${vaGovFullDomain}/img/signin/mhv-icon.svg`}
-        />
-        <strong> Sign in with My HealtheVet</strong>
+        DS Logon
       </button>
       <button
         disabled={isDisabled}
-        className="usa-button-primary va-button-primary"
-        onClick={() => loginHandler('idme')}
+        type="link"
+        aria-label="Sign in with My HealtheVet"
+        className="usa-button usa-button-big mhv-button vads-u-margin-y--1p5"
+        onClick={() => loginHandler('mhv')}
       >
-        <img
-          aria-hidden="true"
-          role="presentation"
-          alt="ID.me"
-          src={`${vaGovFullDomain}/img/signin/idme-icon-white.svg`}
-        />
-        <strong> Sign in with ID.me</strong>
+        My HealtheVet
       </button>
-      <span className="sidelines">OR</span>
       <div className="alternate-signin">
-        <h2 className="vads-u-font-size--sm vads-u-margin-top--0">
-          Don't have those accounts?
-        </h2>
-        <button
-          disabled={isDisabled}
-          className="idme-create usa-button usa-button-secondary"
-          onClick={() => signup('v1')}
-        >
-          <img
-            aria-hidden="true"
-            role="presentation"
-            alt="ID.me"
-            src={`${vaGovFullDomain}/img/signin/idme-icon-dark.svg`}
-          />
-          <strong> Create an ID.me account</strong>
-        </button>
-        <p>Use your email, Google, or Facebook</p>
+        <h2 className="vads-u-margin-top--3">Or create an account</h2>
+        <div className="vads-u-display--flex vads-u-flex-direction--column">
+          {loginGovEnabled && (
+            <a
+              role="link"
+              tabIndex="0"
+              aria-label="Create an account with Login.gov. Navigates to Login.gov website"
+              disabled={isDisabled}
+              className="vads-c-action-link--blue vads-u-border-top--1px vads-u-padding-bottom--2"
+              onClick={() =>
+                signup({ version: 'v1', queryParams: { csp: 'logingov' } })
+              }
+            >
+              Create an account with Login.gov
+            </a>
+          )}
+          <a
+            role="link"
+            tabIndex="0"
+            aria-label="Create an account with ID.me. Navigates to ID.me website"
+            disabled={isDisabled}
+            className="vads-c-action-link--blue vads-u-border-top--1px vads-u-padding-bottom--2 vads-u-border-bottom--1px"
+            onClick={() =>
+              signup({ version: 'v1', queryParams: { csp: 'idme' } })
+            }
+          >
+            Create an account with ID.me
+          </a>
+        </div>
+        {loginGovEnabled && (
+          <a
+            className="vads-u-display--block vads-u-margin-top--2"
+            href="#"
+            target="_blank"
```

Comment:
Links that open in a new window need to announce to screen readers. In this case adding `aria-label="Learn more about choosing an account (Opens in new window)"` would suffice